Sources/MapboxCoreNavigation/EventDetails.swift

```diff
@@ -76,13 +83,9 @@ protocol NavigationEventDetails: EventDetails {
     var driverMode: String { get }
```

Comment:
I guess you can move `driverMode` to `GlobalEventDetails` as well, so we can understand if the departure event is coming from free drive or from active guidance

Reply:
Also `sessionIdentifier` and `startTimestamp `, because you pass `SessionState` to `FreeDriveEventDetails` anyway.

Reply:
`userIdentifier` and  `appMetadata` can be moved as well. Anything except for things specific to feedback: `screenshot`, `feedbackType` and `description`.

Reply:
`GlobalEventDetails` means that all the fields are either have default implementation in the extension or being updated via the extension method. That's why I would rather not move `sessionIdentifier` and `startTimestamp`.
`driverMode` isn't needed for free drive events. The event schema is different under the hood, so there is no `userIdentifier` either.
We can continue this discussion later as it's a private API.